spring-cloud-gcp-starters/pom.xml

```diff
@@ -21,10 +21,7 @@
 	</properties>
 	<modules>
 		<module>spring-cloud-gcp-starter-pubsub</module>
-		<module>spring-cloud-gcp-starter-config</module>
-		<module>spring-cloud-gcp-starter-core</module>
 		<module>spring-cloud-gcp-starter-storage</module>
-		<module>spring-cloud-gcp-starter-sql</module>
```

Comment:
Removing this would break our integration because `DefaultCloudSqlJdbcInfoProvider` depends on `GcpCloudSqlProperties.getDatabaseType()` to return the correct driver class name.

Spring Boot users might find it odd to have to add the MySQL libraries to the classpath _and_ having to specify which kind of SQL database vendor they're using.